package.json

```
@@ -5,7 +5,8 @@
   "main": "index.js",
   "types": "index.d.ts",
   "scripts": {
-    "test": "standard && tap -J test/*.test.js && npm run test:typescript",
+    "testonly": "tap -J test/*.test.js && npm run test:typescript",
+    "test": "standard --fix && tap -J test/*.test.js && npm run test:typescript",
```

It should not have `--fix` parameter for `standard`, because it can not ensure the commit files is linted.
```suggestion
    "test": "standard && tap -J test/*.test.js && npm run test:typescript",
```

---

I added this because on test I was getting lots of small lint errors so it seemed convenient to have `--fix` fix those for me when running test

How do I run `standard --fix` from the command line?